lib/ht/search_client/property_ids_stay_search.rb

```diff
@@ -0,0 +1,83 @@
+module Ht::SearchClient
+  #
+  # GET /places/:id/stays?from=:from&to=:to
+  #
+  # For a given place and dates, return matching stays
+  #
+  # Parameters:
+  #   Required:
+  #     - :property_ids - ids of properties to be checked, in an array
+  #     - :from - the check-in date
+  #     - :to - the check-out date
+  #   Optional:
+  #     - :order - order by
```

Comment:
Yeah, I can't think of any case where you will be passing more than `from, to, guests, and property_ids` @pedrocunha ?